frontend/src/hooks/analytics/useAnalytics.tsx

```diff
@@ -1,3 +1,4 @@
+import logEvent from 'api/common/logEvent';
```

Comment:
why are we importing `logEvent` in the file and exporting it from the hook ? can we directly use it ?

Reply:
Majorly done just to categorize this logEvent under useAnalytics, as its a similar option as trackEvent

Reply:
so if somebody wants to use the logEvent they will have to process the entire hook code. We should keep it separate.

Reply:
Sure, will use logEvent directly here